src/js/lightbeamVR.js

```diff
@@ -0,0 +1,36 @@
+// eslint-disable-next-line no-unused-vars
+const lightbeamVR = {
+  init() {
+    this.loadLightbeamVR();
+  },
+
+  loadLightbeamVR() {
+    const vrView = document.getElementById('vr-view-button');
+    vrView.addEventListener('click', async () => {
+      await this.runLightbeam();
+    });
+  },
+
+  async runLightbeam() {
```

Comment:
I would prefer not to have this code at all. The background script that currently checks for "index.html" could also check for indexVR.
I would load the url in the same page rather than create another tab.